[Datasets] Remove "ML Preprocessing" user guide (#32965)
ray-project/ray: doc/source/ray-air/check-ingest.rst

Shuffling Data
~~~~~~~~~~~~~~

Shuffling or data randomization is important for training high-quality models.

> By default, AIR reads data files (blocks) in a random order and shuffles the blocks
> assigned to each dataset shard. You can disable this behaviour by setting

Reviewer: ```suggestion
By default, AIR shuffles the assignment of data blocks (files) to dataset shards between epochs. You can disable this behavior by setting
```

Reviewer: Also not sure if we should call out how to disable this, since this is a very cheap operation that is an easy win for improving training data shuffling. Is there a use case in which you'd want to disable this? If not, I don't think we need a disabling call out.

Author: If you're evaluating a model on a validation set, you wouldn't want to shuffle data blocks.
